C++/4. Advanced-Operators.md

```diff
@@ -141,12 +148,12 @@ int main() {
 ### Output
 
 ```shell
-0
-1
-1
+false
+true
+true
 ```
 
-**Note: In the Logical and Relational Operators output, 0 means false and 1 means true. These type of operators are mostly not used like this, and are instead used in while and for loops, and in if statements.**
+**Note: These type of operators are mostly not used like this, and are instead used in while loops, for loops, if statements, and in ternary expressions (all of which will be covered later).**
```

Comment:
ternary expressions?? Sorry for my limited knowledge (though I think if you explain it I will understand)

Reply:
Oops, meant "condition operations," C's `?:` operator.

Reply:
oh the if else one right. Understood